### DISCUSSION TITLE
服务器端能不能屏蔽回国流量

### DISCUSSION THREAD

**Discussion by xcf13363175** [emptysuns/Hi_Hysteria#203] [Ideas]
服务器端能不能屏蔽回国流量

**Comment by emptysuns** [comment]
可以
1. 下载geoip`https://cdn.jsdelivr.net/gh/Loyalsoldier/geoip@release/Country.mmdb`，放到`/etc/hihy/acl`里
2. 修改服务端配置文件, `/etc/hihy/conf/hihyServer.json`插入一行关于geoip的配置`"mmdb": "acl/Country.mmdb",`
3. 修改acl匹配cn流量，在`/etc/hihy/acl/hihyServer.acl`文件末尾加上`block country cn`

重启服务端即可

不考虑加入脚本，因为客户端acl会自动`direct country cn`

**Reply by xcf13363175** [reply]
双保险吧，感谢回复。

**Reply by birdeagles** [reply]
我的vps单单加入"acl/Country.mmdb"相对路径不行，必须用绝对路径/etc/hihy/acl/Country.mmdb
